test/e2e/unjoin_e2e/unjoin_test.go

```diff
@@ -0,0 +1,39 @@
+package unjoin_e2e
+
+import (
+	context2 "context"
+	"github.com/liqotech/liqo/test/e2e/util"
+	"gotest.tools/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
+	"testing"
+)
+
+func TestUnjoin(t *testing.T) {
+	context := util.GetTester()
+	NoPods(context.Client1, context.Namespace, t, "cluster1")
```

Comment:
I think I'm not seeing it: where do you begin unjoin process?

Reply:
It is in liqops actions, these tests are run after the uninstall script

Reply:
ok I found it